terraform/state_v1_test.go

```diff
@@ -1,118 +1,147 @@
 package terraform
 
 import (
+	"bufio"
 	"bytes"
-	"encoding/gob"
-	"errors"
-	"io"
 	"reflect"
-	"sync"
+	"strings"
 	"testing"
 
-	"github.com/mitchellh/hashstructure"
+	"github.com/davecgh/go-spew/spew"
```

Comment:
Just checked - spew is still vendored on the `maint-0.6` branch so we should be :+1: here.